daprdocs/content/en/developing-applications/building-blocks/observability/w3c-tracing/w3c-tracing-howto.md

```diff
@@ -135,7 +135,7 @@ You can create a trace context using the recommended OpenCensus SDKs. OpenCensus
 | Language | SDK |
 |:-------:|:----:|
 | Go | [Link](https://pkg.go.dev/go.opencensus.io?tab=overview)
-| Java | [Link](https://www.javadoc.io/doc/io.opencensus/opencensus-api/latest/index.html)
+| Java | [Link](https://www.javadoc.io/doc/io.opencensus/opencensus-api)
```

Comment:
I cannot get this link to work. How about this link https://github.com/census-instrumentation/opencensus-java instead